samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -22,9 +22,8 @@ public partial class DocumentTranslationClient
         private const string AuthorizationHeader = "Ocp-Apim-Subscription-Key";
         private readonly AzureKeyCredential _keyCredential;
         private readonly HttpPipeline _pipeline;
-        private readonly ClientDiagnostics _clientDiagnostics;
         private readonly string _endpoint;
-
+        internal ClientDiagnostics ClientDiagnostics { get; }
         /// <summary> The HTTP pipeline for sending and receiving REST requests and responses. </summary>
         public virtual HttpPipeline Pipeline => _pipeline;
```

Comment:
nit: skip line

Reply:
There is no empty line here. It's diff display.

Reply:
I meant can we add an empty line? like we had before(see [this](https://github.com/Azure/autorest.csharp/blob/feature/v3/samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs#L27)) after the private members. The code looks much cleaner this way.